examples/gallery/ui/pages/table_view_page.slint

```diff
@@ -0,0 +1,83 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { HorizontalBox, VerticalBox, StandardTableView, GroupBox} from "std-widgets.slint";
+import { GallerySettings } from "../gallery_settings.slint";
+import { Page } from "page.slint";
+
+export global TableViewPageAdapter := {
+    callback sort_ascending(int);
+    callback sort_descending(int);
+    property <[StandardTableRow]> row_data;
+}
+
+export TableViewPage := Page {
+    title: "TableView";
+    description: "StandardListBox can be used to display a list of text elements in columns and rows. It can be imported from \"std-widgets.slint\"";
+
+    HorizontalBox {   
+        vertical-stretch: 1;
+
+        GroupBox {  
+            title: "StandardTableView";
+            vertical-stretch: 0;
+
+            StandardTableView {
+                private property <int> current-sort-column: -1;
+
+                column-clicked(index) => {
+                    if (current-sort-column != index) {
+                        columns[current-sort-column].sort-by-ascending = false;
+                        columns[current-sort-column].sort-by-descending = false;
+                    }
+
+                    if(columns[index].sort-by-ascending) {
+                        columns[index].sort-by-ascending = false;
+                        columns[index].sort-by-descending = true;
+                        TableViewPageAdapter.sort_descending(index);
+                    } else {
+                        columns[index].sort-by-ascending = true;
+                        columns[index].sort-by-descending = false;
+                        TableViewPageAdapter.sort_ascending(index);
+                    }
+
+                    current-sort-column = index;
+                }
+                property <StandardTableRow> first-row: {
```

Comment:
What are these properties for? Is that left-over dummy data?
I think it makes sense to have dummy data anyway, but probably in a ` [{items: [...]}, {items: [...]}, ...]`

Reply:
Yes I now it's a workaround because `[{items: [...]}, {items: [...]}, ...]`  does not work at the moment in Slint

Reply:
Ok but yes this data is no longer needed.

Reply:
removed

Reply:
> Yes I now it's a workaround because [{items: [...]}, {items: [...]}, ...] does not work at the moment in Slint

I think https://github.com/slint-ui/slint/pull/2025 should make it work.